assets/js/app.js

```diff
@@ -344,6 +344,18 @@ let liveSocket = new LiveSocket("/live", Socket, {
   }
 })
 
+Hooks.EmojiButtonHook = {
+  mounted() {
+    const picker = new EmojiButton();
+    const trigger = document.querySelector("#emoji-button");
```

Comment:
Our next step is to abstract the id of the element out of there. We can have multiple emoji buttons on one page (imagine a page with several text areas that have emoji keyboards). We need a way for the `mounted()` callback to know the element it corresponds to, and to communicate the input element we want to be adding the emoji to from the backend.